test/agency.js

```diff
@@ -0,0 +1,59 @@
+const config = require('../config');
```

Comment:
This is a whole test setup question. Would you be opposed to it being agency_tests.js or something like that so at a general glance you know this is tests and not someone duplicating or misplacing. Completely a non-issue just question/what not.

Reply:
I considered it but thought it might be redundant (`tests/agency_test.js`)